templates/flavors/kubeadm/dns-loadbalancing/kustomization.yaml

```diff
@@ -20,3 +20,55 @@ patches:
           dnsUniqueIdentifier: ${DNS_UNIQUE_ID}
           dnsProvider: ${DNS_PROVIDER:-"linode"}
           dnsSubDomainOverride: ${DNS_SUBDOMAIN_OVERRIDE:-""}
+  - target:
+      group: infrastructure.cluster.x-k8s.io
+      version: v1alpha2
+      kind: LinodeFirewall
+    patch: |-
+      apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+      kind: LinodeFirewall
+      metadata:
+        name: ${CLUSTER_NAME}
+      spec:
+        inboundRules:
+          - action: ACCEPT
+            label: intra-cluster-tcp
+            ports: "1-65535"
+            protocol: "TCP"
+            addresses:
+              ipv4:
+                - "10.0.0.0/8"
+            description: accept all tcp traffic within the vpc
+          - action: ACCEPT
+            label: intra-cluster-udp
+            ports: "1-65535"
+            protocol: "UDP"
+            addresses:
+              ipv4:
+                - "10.0.0.0/8"
+            description: accept all udp traffic within the vpc
+          - action: ACCEPT
+            label: intra-cluster-icmp
+            protocol: "ICMP"
+            addresses:
+              ipv4:
+                - "10.0.0.0/8"
+            description: accept all icmp traffic within the vpc
+          - action: ACCEPT
+            addresses:
+              ipv4:
+                - 0.0.0.0/0
+              ipv6:
+                - ::/0
+            ports: "${APISERVER_PORT:=6443}}"
```

Comment:
for kubeadm, do we need to add konnectivity port here?

Reply:
If we expect dns-loadbalancing flavor to not have konnectivity support, then we should be good here

Reply:
I don't think we have any flavor that does konnectivity + DNS loadbalancing right now, if we added one we could include this port information as well